messages/jsonschema/GherkinDocument.json

```diff
@@ -347,8 +348,20 @@
           "description": "The location of the steps' `keyword`"
         },
         "keyword": {
+          "description": "The actual keyword as it appeared in the source.",
           "type": "string"
         },
+        "keywordType": {
+          "description": "The test phase signalled by the keyword: Contex definition (Given), Action performance (When), Outcome assertion (Then). Other keywords signal Continuation (And and But) from a prior keyword. Please note that all translations which a dialect maps to multiple keywords (`*` is in this category for all dialects), map to 'Unknown'.",
+          "type": "string",
+          "enum": [
```

Comment:
other enums have the 'default' value in the first position. This would make more sense with Unknown first IMO, as we could use that rather than null as a sane default

Reply:
Changed. (And changed the typo on `Contex` above too as you noted in the Slack channel.)